### ISSUE TITLE
Ubuntu devel builder does not work on devel branch

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#2222]
The job `Build-Tribler_Ubuntu-64_devel` is currently not working and failing on the `dch` command. On next, it works. The problem appears to be due to the fact that the result of `git describe --tags HEAD` is different on `next` (`v6.5.2`) than on `devel` (`v6.5.0-rc5-772-g658647a`).

@whirm what do you suggest how to fix this? Should we split on `-` character and take the first part?

**Comment by synctext** [comment]
Error link:
https://jenkins.tribler.org/job/Build-Tribler_Ubuntu-64_devel/584/console

```
+ Tribler/Main/Build/update_version_from_git.py
Traceback (most recent call last):
  File "Tribler/Main/Build/update_version_from_git.py", line 43, in <module>
    runCommand('dch -v {} New upstream release.'.format(version_id).split())
  File "Tribler/Main/Build/update_version_from_git.py", line 17, in runCommand
    assert(p.returncode == 0)
AssertionError
```

Failing code:
https://github.com/Tribler/tribler/blob/devel/Tribler/Main/Build/update_version_from_git.py#L43

```runCommand('dch -v {} New upstream release.'.format(version_id).split())```

**Comment by synctext** [comment]
@devos50  Now to actually answer your question:-)
Yes, please do ```split("-")[:1]```. Then we can start testing

**Comment by whirm** [comment]
Let's just add a new tag

**Comment by devos50** [comment]
@whirm that's even easier but for easier future releases and stable automated building, we can still do the split thing?

**Comment by devos50** [comment]
@whirm @synctext Ubuntu builder working 👍 

See the fresh `.deb` files here: https://jenkins.tribler.org/job/Build-Tribler_Ubuntu-64_devel/lastSuccessfulBuild/artifact/

**Comment by whirm** [comment]
The issue here is that the changelog has a >6.5.0 release registered, so dch refuses to add a lower version after it. stripping part of the version would not fix anything (6.5.0 would still be lower than 6.5.2) and would be misleading (6.5.0 is not the same than 6.5.0+700 commits).
